### PR TITLE
Update main-da.json

### PR DESCRIPTION
**Pull request by palle64**
Improved the Danish translation of Jitsi

### CODE REVIEW

**Comment by palle64** [comment]
> Hi, thanks for your contribution!
> If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

I have done that now :-)

**Review by jokjr — CHANGES_REQUESTED** [review]
I have some suggestions :)

File: lang/main-da.json [hunk 1]
```diff
@@ -295,46 +296,46 @@
...
-        "cancelPassword": "Annulé adgangskode",
+        "cancelPassword": "Annullér adgangskode",
         "conferenceURL": "Link:",
         "country": "Land",
         "dialANumber": "For at deltage i mødet ring til et af følgende telefonnumre og indtast pinkode.",
         "dialInConferenceID": "Pinkode:",
-        "dialInNotSupported": "Deltagelse vis telfonen er pt. ikke understøttet.",
+        "dialInNotSupported": "Deltagelse via telfonen er pt. ikke understøttet.",
         "dialInNumber": "Ring ind:",
         "dialInSummaryError": "Der er opstået en fejl under hentning af ring ind information. Forsøg venligst igen senere.",
```
[the remark this answers is not in the document]
Comment (palle64) [reply lang/main-da.json:320]: Jeg var også med i tvivl ang. denne oversættelse, men jeg synes dit forslag er godt, så det retter jeg det lige til.

**Review by jokjr — APPROVED** [review]

**Comment by damencho** [comment]
Jenkins, test this please.

**Review by damencho — APPROVED** [review]

**Pull request merged**